contracts/feature-tests/basic-features/src/lib.rs

```diff
@@ -878,6 +878,23 @@ pub trait BasicFeatures {
 		self.keccak256(&input)
 	}
 
+	// Not called, they currently just panic with "Not implemented yet!"
+
+	#[endpoint(verifyBlsSignature)]
```

Comment:
you can leave the Rust method name to be the endpoint name here, we only use this contract for tests, it's easier to follow this way. I think the other endpoints are snake case too